enacts/config-test-enacts.yaml

```diff
@@ -76,6 +76,24 @@ datasets:
     # Soil data
     taw_file: /data/remic/mydatafiles/soilgrids/Senegal/GYGA_ERZD_wat_cap_abs_anacim_enacts3.nc
 
+    # Forecast data
+    fcst_data:
+        #forecast_path: /data/remic/mydatafiles/ANACIM/cpt_demo/UCSBV3
+        forecast_path: /data/remic/mydatafiles/ANACIM/cpt_demo/UCSBV3/Jul-Sep
```

Comment:
Use one or the above to test the nc multiple/single cases. Everything else below pertains to tsv case.

Reply:
Put that comment in the file.